### ISSUE TITLE
[BUG] error when handling missing tumor-TIL pairs 

### ISSUE THREAD

**Issue by kaczmarj** [SBU-BMI/til_align#22]
@lthealy - i am getting an error when testing this with missing tumor/til pairs. when you fix, can you test with missing pairs?

```r
[1] "TIL Algorithm (Threshold): Frontiers InceptionV4: 0.1"
[1] "=========== Params after R parsing, if any misalignment please check your flags ==========="
$algorithm
[1] "inceptionv4"

$tilDir
[1] "/data/results-tils"

$tilThresh
[1] 0.1

$cancDir
[1] "/data/results-tumor"

$cancThresh
[1] 0.5

$sampFile
[1] ""

$outputFile
[1] "output.csv"

$outputDir
[1] "/data/results-tilalign/"

$writePNG
[1] TRUE

$sampInfo
[1] "/data/sample_info.csv"

 . . . Dropping low_res and color- files . . . 
 . . . Checking for tumor/lymph pairs . . . 
Error in length(setdiff(tils, canc), " out of ", length(tils), " are missing pairs. They will be dropped") : 
  4 arguments passed to 'length' which requires 1
Calls: writeLines -> paste0
In addition: Warning message:
Some cancer-lymph predictions are missing pairs, skipping non-paired samples 
Execution halted
```

**Comment by lthealy** [comment]
Added dummy missing pairs for debugging, new commit and PR runs locally with no issue

**Comment by lthealy** [comment]
Fixed with PR#23
